internal/presenters/components.go

```diff
@@ -8,9 +8,58 @@ import (
 	"strings"
 	"text/template"
 
+	"github.com/charmbracelet/lipgloss"
+	"github.com/snyk/error-catalog-golang-public/snyk_errors"
 	"github.com/snyk/go-application-framework/pkg/local_workflows/json_schemas"
 )
 
+func errorLevelToStyle(errLevel string) lipgloss.Style {
+	style := lipgloss.NewStyle().
+		PaddingLeft(1).
+		PaddingRight(1).
+		Background(lipgloss.Color("1")).
+		Foreground(lipgloss.Color("15"))
+
+	if errLevel == "warn" {
+		style.
+			Background(lipgloss.Color("3")).
+			Foreground(lipgloss.Color("0"))
+	}
+
+	return style
+}
+
+func RenderError(err snyk_errors.Error) string {
+	level := strings.ToUpper(err.Level)
+
+	backgroundHighlight := errorLevelToStyle(err.Level)
+
+	var body []string
+
+	if len(err.Detail) > 0 {
+		infoLabel := lipgloss.NewStyle().Width(8).Render("Info:")
```

Comment:
Suggestion: For all values like Width, padding ... that are used in multiple places it makes sense to introduce a constant as a single source of truth.

Reply:
Removed wrapping for link block https://github.com/snyk/go-application-framework/pull/194/commits/e397089af9096bb5991d22c6b54fa88a620df858

Reply:
Addressed in https://github.com/snyk/go-application-framework/pull/194/commits/cae6cfa85c5539abdabf705c177f8057abddb79c